DEV_GUIDE.md

```diff
@@ -0,0 +1,214 @@
+# Development guide
+
+This guide explains how to modify or contribute to the Forjj  project.
+Forjj is written in GO language, that needs to be compiled. This is the main difficulty, however some scripts have been written to really simplify this process.
+
+
+## Repositories
+
+To properly build Forjj you need to get the sources from github.
+
+Forjj is composed of various repositories that are available from the project main page https://github.com/forj-oss .
+- forjj --> main program, cli
+- forjj-contribs --> plugins (github, jenkins)
+- forjj-flows --> ?
```

Comment:
This repository defines Project development flows that forjj can configure in a new or existing forge. 
It will contain some description files to build the interface between applications (through forjj plugins) which will support the development workflow.

Ex: flow 'pull_request' will create the required links between github webhook and jenkins to start jenkins on Pull request.